src/main/java/org/traccar/forward/MqttClient.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2023 Anton Tananaev (anton@traccar.org)
```

Comment:
Please fix the year in notices. Also if you want you can change the name and email to your own.